controllers/propagator/policy_controller.go

```diff
@@ -5,6 +5,7 @@ package propagator
 
 import (
 	"context"
+	genericError "errors"
```

Comment:
This was confusing to me. Could you please keep this named errors and rename `k8s.io/apimachinery/pkg/api/errors` to be `k8serrors`? Though, I am asking you to remove the one use of the `errors` package, so this can be disregarded if you agree with that.